### PR TITLE
Improve Testing

### PR DESCRIPTION
**Pull request by Myoldmopar**

### CODE REVIEW

File: eplaunch/tests/utilities/energyplus [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "EnergyPlus, Version 5.9.0-deadbeef00"
```
Comment (Myoldmopar) [inline eplaunch/tests/utilities/energyplus]: I created a dummy script that will just echo the version in the same format E+ does

File: eplaunch/tests/utilities/test_locateworkflows.py [hunk 2]
```diff
@@ -37,17 +38,34 @@ def test_locate_workflow_directories(self):
...
+
+        loc_wf.get_energyplus_versions()
+        self.assertEqual(1, len(loc_wf.list_of_energyplus_versions))
+        self.assertEqual('5.9.0', loc_wf.list_of_energyplus_versions[0]['version'])
+        self.assertEqual('deadbeef00', loc_wf.list_of_energyplus_versions[0]['sha'])
+
     def tearDown(self):
         try:
-            os.rmdir(self.workflow_directory)
+            shutil.rmtree(self.workflow_directory)
```
Comment (Myoldmopar) [inline eplaunch/tests/utilities/test_locateworkflows.py]: `rmtree` can handle non-empty folders, which is the case now that I am putting in the little mock energyplus file

File: eplaunch/utilities/locateworkflows.py [hunk 3]
```diff
@@ -31,13 +33,21 @@ def find(self):
         return self.list_of_found_directories
 
     def get_energyplus_versions(self):
+        ep_names = {
+            Platform.WINDOWS: 'energyplus.exe',
+            Platform.LINUX: 'energyplus',
+            Platform.MAC: 'energyplus',
+            Platform.UNKNOWN: ''
+        }
```
Comment (Myoldmopar) [inline eplaunch/utilities/locateworkflows.py]: Added a dictionary for getting the right binary name here

File: eplaunch/utilities/locateworkflows.py [hunk 4]
```diff
@@ -31,13 +33,21 @@ def find(self):
...
         for workspace_directory in self.list_of_found_directories:
             energyplus_directory, folder_name = os.path.split(workspace_directory)
-            energyplus_application = os.path.join(energyplus_directory, "energyplus.exe")
+            energyplus_application = os.path.join(energyplus_directory, ep_names[Platform.get_current_platform()])
             if os.path.exists(energyplus_application):
                 self.list_of_energyplus_applications.append(energyplus_application)
                 found, version_string, build_string = self.get_specific_version_from_exe(energyplus_application)
-                if not found:
+                if found:
+                    self.list_of_energyplus_versions.append({'version': version_string, 'sha': build_string})
```
Comment (Myoldmopar) [inline eplaunch/utilities/locateworkflows.py]: I'm not **actually** sure what you wanted to store for each version, but I figured a little dictionary would suffice for now.  It allows testing it anyway.

**Comment by JasonGlazer** [comment]
Thanks. Are you still working on this branch or should I merge it?

**Comment by Myoldmopar** [comment]
Still working.  While I'm on my other phone calls, just multitasking.

**Pull request merged**